### PR TITLE
Add images with popups to Claim cards and Claim view

### LINKED ISSUE #390: Update the ViewClaim Page to contain new proof information 3
It must have a modal for showing the picture in more details

### PR DESCRIPTION
**Pull request by amiskov**
## What issue does this PR close
Closes #390 #392. Frontend only (placeholder used for the photo).

## Changes Proposed (a list of new changes introduced by this PR)
Add all the logic and design changes for the new claims according to the designs (see [Analyst's Flow](https://www.figma.com/file/N2lbB9xdO6tFbfg2Is4h7s/claim_with_image?node-id=2%3A972) screens).

There are several TODOs for which I need some extra stuff. I'll get back to them after merging this branch into `new_claims` and implementing the backend part.

## How to test (a list of instructions on how to test this PR)
There are a couple of examples with the placeholders in the `0,AMTA` community for the analyzer `amiskovstag4`. There should be 2 Claim Cards with the photo on the Dashboard and Analysis page. Photos are placeholders, but voting buttons should work properly.

### CODE REVIEW

File: src/elm/Claim.elm [hunk 1]
```diff
@@ -291,19 +359,66 @@ viewClaimCard { selectedCommunity, shared, accountName } openConfirmationModalMs
...
+                    [ label [ class "mt-6 md:mt-0 input-label md:text-xl block" ]
+                        [ text "verification number"
+                        ]
+                    , strong [ class "text-xl md:text-3xl" ] [ text "82378463" ]
+                    ]
+                ]
+            ]
+
+        withPhotoModalFooter =
+            if isValidated claim loggedIn.accountName then
```
Comment (lucca65) [inline src/elm/Claim.elm:394]: we shouldn't show the buttons if the user is not a validator as well, this way anyone can check the status of the claim, even if its not a validator

**Review by lucca65 — CHANGES_REQUESTED** [review]
Just one little check, that we need to be sure in all interactions with the claim page. It should be public, everyone could see the status of a claim, but only validators can vote

File: src/elm/Claim.elm (hunk 1 above)
Comment (amiskov) [reply src/elm/Claim.elm]: Yes, indeed! I've done it on the View Claim but forgot to add here... Should be fixed now!

**Review by lucca65 — APPROVED** [review]
👌🚀

**Pull request merged**